randovania/gui/game_details/game_details_window.py

```diff
@@ -250,16 +250,22 @@ async def _open_map_tracker(self):
 
     # Layout Visualization
 
-    def update_layout_description(self, description: LayoutDescription):
+    def update_layout_description(self, description: LayoutDescription, players: list[str] | None = None):
         self.layout_description = description
         self.layout_info_tab.show()
 
         self.setWindowTitle(f"Game Details: {description.shareable_word_hash}")
         self.export_log_button.setText("Save Spoiler" if description.has_spoiler
                                        else "Save to file")
 
+        if players is None:
+            def _get_player_name(i):
```

Comment:
can't this be done differently? like, set `players` to something if None, then `self._player_names = dict(enumerate(players))`?

Reply:
sure, i was rushing this for sure lol